easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PackedBinary'
+
+name = 'Lucene-Geo-Gazetteer'
+version = '20170718' 
+
+homepage = 'https://github.com/chrismattmann/lucene-geo-gazetteer'
+description = """A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library
+ to create a searchable gazetter."""
```

Comment:
@boegel Please use the same spelling of gazetteer within the same sentence...

Reply:
Copy-paste from the README at https://github.com/chrismattmann/lucene-geo-gazetteer, but I guess it's a typo... I'll fix it, also sent them a PR: https://github.com/chrismattmann/lucene-geo-gazetteer/pull/26